pennylane/devices/_qutrit_device.py

```diff
@@ -183,6 +183,24 @@ def vn_entropy(self, wires, log_base):
             "Unsupported return type specified for observable Von Neumann entropy"
         )
 
+    def vn_entanglement_entropy(self, wires0, wires1, log_base):
```

Comment:
Same question here.  Do we actually need this method for any reason?

Reply:
See above.